web/src/assets/locales/en.json

```diff
@@ -526,5 +526,49 @@
       "desc": "Pipelines created by the listed users never require approval."
     }
   },
-  "no_search_results": "No results found"
+  "no_search_results": "No results found",
+  "forges": "Forges",
+  "forges_desc": "Configure Forges hosting repositories Woopecker should run for.",
+  "add_forge": "Add forge",
+  "show_forges": "Show forges",
+  "github": "GitHub",
+  "gitlab": "GitLab",
+  "bitbucket": "Bitbucket",
+  "bitbucket_dc": "Bitbucket Data Center",
+  "gitea": "Gitea",
+  "forgejo": "Forgejo",
+  "addon": "Addon-Forge",
+  "forge_type": "Forge type",
+  "oauth_client_id": "OAuth Client ID",
+  "oauth_client_secret": "OAuth Client Secret",
+  "oauth_host": "OAuth Host",
+  "merge_ref": "Merge ref",
+  "merge_ref_desc": "Ref to use for merge base. This is used to determine the diff for pull requests.",
+  "public_only": "Public only",
+  "public_only_desc": "Only show public repositories",
+  "git_username": "Git username",
+  "git_username_desc": "Username for the Git user",
+  "git_password": "Git password",
+  "git_password_desc": "Password or personal access token for the Git user",
+  "executable": "Executable",
+  "executable_desc": "Path to the addon executable",
+  "save": "Save",
+  "add": "Add",
+  "skip_verify": "Skip SSL verification",
+  "skip_verify_desc": "Skip SSL verification for the api connection. This is NOT recommended for production use.",
+  "url": "URL",
+  "forge_managed_by_env": "The primary forge is managed by environment variables. Any changes to this forge will be reverted on a restart.",
+  "oauth_redirect_uri": "OAuth redirect URI",
+  "forge_created": "Forge created",
+  "advanced_options": "Advanced options",
+  "leave_empty_to_keep_current_value": "Leave empty to keep current value",
+  "forge_deleted": "Forge deleted",
+  "forge_delete_confirm": "Do you really want to delete this forge? This will also delete all repositories, users, pipelines related to this forge.",
```

Comment:
```suggestion
  "forge_delete_confirm": "Do you really want to delete this forge? This will also delete all repositories, users and pipelines related to this forge.",
```